docs/pages/corefunctionality/sources.rst

```diff
@@ -8,6 +8,7 @@ There are a few different potential sources for Simulations:
 .. code-block:: java
 
     SimulationSource.classpath("simulation.json"); //classpath
+    SimulationSource.classpathBelowHoverflyDir("simulation.json"); //classpath prefixed with /hoverfly
```

Comment:
This simulation source name is confusing. How about `defaultPath("simulation.json")`